src/v/cloud_storage/tests/remote_partition_fuzz_test.cc

```diff
@@ -27,6 +27,7 @@ scan_remote_partition_incrementally_with_reuploads(
   std::vector<in_memory_segment> segments,
   size_t maybe_max_segments = 0,
   size_t maybe_max_readers = 0) {
+    ss::lowres_clock::update();
```

Comment:
i thought we could patch seastar to fix this

Reply:
also, doesn't this suggest that our code is not robust?

Reply:
this is a temporary measure, the problem only appears in some unit-tests (in a very annoying way) so this is why temporary fix is needed